meson.build

```diff
@@ -4,8 +4,8 @@ project('classdesc', 'cpp',
     meson_version: '>=0.39.1',
     license: [ 'GPL3+', 'MIT' ],
     default_options: [
-        'cpp_std=c++03',
```

Comment:
I didn't realise there was a C++03 on the table - that would have done, as it is not significantly different from C++98. The main thing is that it compiles on pre-modern  (ie prior to c++11) standard compilers.

Still odd that meson doesn't provide a c++98, as that is the first, and still best known standard.

Also, all that is required is a global compiler option -std=c++98. That shouldn't need patching meson.